schema/spacewalk/common/tables/rhnActionClusterGroupRefreshNodes.sql

```diff
@@ -0,0 +1,31 @@
+-- Copyright (c) 2020 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE rhnActionClusterGroupRefreshNodes (
+    action_id           NUMERIC NOT NULL
+                            CONSTRAINT rhn_actionclgrref_aid_fk
+                            REFERENCES rhnAction (id)
+                            ON DELETE CASCADE,
+    cluster_id          NUMERIC
+                            CONSTRAINT rhn_actionclgrref_cluster_fk
+                            REFERENCES suseClusters (id)
+                            ON DELETE CASCADE,
+    created             TIMESTAMPTZ
+                            DEFAULT (CURRENT_TIMESTAMP) NOT NULL,
+    modified            TIMESTAMPTZ
+                            DEFAULT (CURRENT_TIMESTAMP) NOT NULL
```

Comment:
Is `modified` needed or used at all?
You do not have a trigger configured and I am unsure if the java hibernate part is adapting it automatically.
AFAIK not as this was only possible with annotations. 

This apply to all these Action classes.